<?xml version="1.0"?>
<launch>
	<!-- Input arguments -->
	<arg name="argID"               default="0"/>
	<arg name="argNodeRate"         default="100.0"/>
	<arg name="argNamespace"        default="heifu$(arg argID)"/>
	<arg name="argVehicle"          default="$(arg argNamespace)"/>
    <arg name="argSafetyDistance"   default="1.0"/>

	<node name="CollisionAvoidance" type="collision_avoidance_node" pkg="collision_avoidance" output="screen" required="true" ns="$(arg argNamespace)">
		<param name="paramNodeRate"         type="double" value="$(arg argNodeRate)"		/>
		<param name="paramSafetyDistance"	type="double" value="$(arg argSafetyDistance)"	/>
		<rosparam param="paramWorkspaceSize">[20.0, 20.0, 10.0]</rosparam>
	</node>
</launch>
